juju/juju
# Add new k8s-spec-set and k8s-spec-get commands

## Description of change

The pod-spec-set/get commands are deprecated in favour of k8s-spec-set/get.
Also, the previous work to rename actions back to functions left some cruft so this was fixed.

## QA steps

deploy a k8s charm
exec into the operator pod and ensure that k8s-spec-get/set commands and the pod-spec versions are present
juju-run both pod-spec-get and k8s-spec-get
edit a local charm to use k8s-spec-set instead of pod-spec-set and deploy
